rfc8446.xml

```diff
@@ -2644,7 +2651,7 @@ containing a single certificate and set of extensions.</t>
   A set of extension values for the CertificateEntry. The "Extension"
 format is defined in <xref target="extensions"/>. Valid extensions for server
 certificates at present include the OCSP Status extension <xref
-target="RFC6066"/> and SignedCertificateTimestamp <xref target="RFC6962"/>;
+target="RFC6066"/> and SignedCertificateTimestamp extension <xref target="RFC6962"/>;
```

Comment:
Do we also need to add "the" for proper parallelism?

Reply:
I can do that.